src/Arachne.cc

```diff
@@ -680,6 +667,7 @@ dispatch() {
             DispatchTimeKeeper::numThreadsRan++;
             return;
         }
+        currentIndex++;
```

Comment:
I think `for (;; currentIndex++)` would be better for clarity.

Reply:
This has been done. I originally had `continue` statements which skipped the increment, but that is no longer applicable.